pkg/common.go

```diff
@@ -0,0 +1,109 @@
+package pkg
+
+import (
+	"strings"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
```

Comment:
Better to keep import order and format files with gofmt

Reply:
Updated.